HW8/08-02-Practical-re.py

```diff
@@ -0,0 +1,13 @@
+import re
+
+def check(paswd: str) -> bool:
+    re_pattern = r"(?=.*[#$@]+)(?=.*[a-z]+)(?=.*[0-9]+)(?=.*[A-Z]+)"
+    if re.search(re_pattern, paswd) and len(paswd) >= 6 and len(paswd) <= 16:
```

Comment:
we can return just 
bool(re.search(re_pattern, paswd) and len(paswd) >= 6 and len(paswd) <= 16)